fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/cascades/values/simplification/ValueSimplificationTest.java

```diff
@@ -367,42 +365,6 @@ void testDeriveOrderingValues() {
         Assertions.assertEquals(expectedResult, orderingValues);
     }
 
-    @Test
```

Comment:
Can you leave the test in here and just remove the `@Test` and add a TODO to put it back in?

Reply:
Or leave this test in here but use a specifically constructed rules set including that rule just for this test case.